logrotation/startup/entrypoint_cron.py

```diff
@@ -0,0 +1,73 @@
+# /usr/bin/python3
+
+import argparse
+import sys
+from cortx.utils.conf_store import Conf
+from cortx.utils.process import SimpleProcess
+import os
+
+sys.path.append('/opt/cortx/component/test')
+from logger import start_logging
+
+
+def get_local(config_url):
+    """Get local log path from the config url."""
```

Comment:
no local path necessarily here, can be shared / remote mount point too

Reply:
removed this function Chetan. 👍